asset-transfer-basic/application-gateway-java/src/main/java/App.java

```diff
@@ -231,20 +231,11 @@ private void updateNonExistentAsset() {
 
 			System.out.println("******** FAILED to return an error");
 		} catch (EndorseException | SubmitException | CommitStatusException e) {
-			System.out.println("*** Successfully caught the error: ");
+			System.out.println("*** Successfully caught the error:");
 			e.printStackTrace(System.out);
 			System.out.println("Transaction ID: " + e.getTransactionId());
-
-			var details = e.getDetails();
-			if (!details.isEmpty()) {
-				System.out.println("Error Details:");
-				for (var detail : details) {
-					System.out.println("- address: " + detail.getAddress() + ", mspId: " + detail.getMspId()
-							+ ", message: " + detail.getMessage());
-				}
-			}
```

Comment:
So glad we don't have to traverse the details now!

Reply:
You can still access them programmatically (as before) if you need to buy this gives a much smoother user experience when things go wrong. I'm wondering if there is anything that can be done better for Go too.